### PR TITLE
Ad9208 jenkins update

### PR DESCRIPTION
**Pull request by SrikanthPagadarai**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -64,6 +64,34 @@ cstage("HDL Tests", "", flags) {
...
+            }
+            finally {
+                cleanWs();
+            }
+        }
+    }
+}}
+
+/////////////////////////////////////////////////////
+
```
Comment (tfcollins) [inline Jenkinsfile:94]: Missing the call to actual run the closures. Need to add
```
parallel deployments
```

File: Jenkinsfile [hunk 2]
```diff
@@ -64,6 +64,34 @@ cstage("HDL Tests", "", flags) {
...
+def deployments = [:];
+def board = 'ad9208';
+def nodeLabel = 'baremetal && high_memory';
+deployments[board] = { node(nodeLabel) {
+    cstage("Baremetal HDL Test", "", flags) {
+        withEnv(['BOARD='+board,'MLRELEASE=R2022b','HDLBRANCH=hdl_2021_r2','LC_ALL=C.UTF-8','LANG=C.UTF-8']) {
+            try {
+                stage("AD9208 HDL Test") {
```
Comment (tfcollins) [inline Jenkinsfile:78]: Please use the cstage API instead of stage

File: Jenkinsfile (hunk 1 above)
Comment (SrikanthPagadarai) [reply Jenkinsfile:92]: Fixed by latest commit.

File: Jenkinsfile (hunk 2 above)
Comment (SrikanthPagadarai) [reply Jenkinsfile]: Fixed by latest commit.

**Pull request merged**